scripts/analysis/compare_terrain_loss.py

```
@@ -0,0 +1,190 @@
+
+def extract_terrain_from_raw_response(raw_response: str) -> List[str]:
+    """Extract terrain from raw_response JSON string."""
+    try:
+        # Find the JSON part in the raw response
+        start = raw_response.find("{")
+        end = raw_response.rfind("}") + 1
+        if start != -1 and end != -1:
+            json_str = raw_response[start:end]
+            data = json.loads(json_str)
+            return data.get("terrain", [])
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)